obspy/imaging/waveform.py

```diff
@@ -37,6 +40,10 @@
 from obspy.core.util import (FlinnEngdahl, createEmptyDataChunk,
                              locations2degrees)
 from obspy.core.util.decorator import deprecated_keywords
+from obspy.imaging.util import ObsPyAutoDateFormatter
+
+
+MINMAX_ZOOMLEVEL_WARNING_TEXT = "Warning: Zooming into MinMax Plot!"
```

Comment:
Ok, I see. I thought it's enough and the text can't be too long. Maybe I'll try to be more clear..